server: TestLogGC failed

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/0de4db45-492c-4d76-9408-79e95f936c10?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [49cff91f3501494deaf038671bc643c194a0e3ca](https://github.com/cockroachdb/cockroach/commits/49cff91f3501494deaf038671bc643c194a0e3ca):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC1257559356
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-14T09:00:33.563429Z/1029231193567035393/0 @1734166833.584263160,0): "sql txn" meta={id=0f7547a8 key=/Table/13/1/2024-12-14T08:55:33.554004Z iso=Serializable pri=0.01844419 epo=0 ts=1734166891.088486312,2 min=1734166553.323012032,0 seq=6000} lock=true stat=PENDING rts=1734166553.323012032,0 wto=false gul=1734166553.823012032,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 14, 9, 55, 33, 554003653, time.UTC)
        	            	actual  : time.Date(2024, time.December, 14, 8, 55, 33, 554003653, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-14 09:55:33.554003653 +0000 UTC m=+3669.965576233
        	            	+(time.Time) 2024-12-14 08:55:33.554003653 +0000 UTC m=+69.965576233
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC1257559356
--- FAIL: TestLogGC (372.01s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45608

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/dea60993-0098-4d0d-aed5-2915d429197e?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [fe048f6e72edca59b4367463db1d0d00783f08ab](https://github.com/cockroachdb/cockroach/commits/fe048f6e72edca59b4367463db1d0d00783f08ab):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC2324780221
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-15T08:47:49.259574Z/1029511804601565185/0 @1734252469.275662033,0): "sql txn" meta={id=e805b044 key=/Table/13/1/2024-12-15T08:42:49.006301Z iso=Serializable pri=0.01190021 epo=0 ts=1734252523.096822283,2 min=1734252187.817121724,0 seq=6000} lock=true stat=PENDING rts=1734252187.817121724,0 wto=false gul=1734252188.317121724,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 15, 9, 42, 49, 6301298, time.UTC)
        	            	actual  : time.Date(2024, time.December, 15, 8, 42, 49, 6301298, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-15 09:42:49.006301298 +0000 UTC m=+3633.226287980
        	            	+(time.Time) 2024-12-15 08:42:49.006301298 +0000 UTC m=+33.226287980
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC2324780221
--- FAIL: TestLogGC (363.83s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/f9012730-be92-45bb-a41b-a0e08397516b?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [9354770c7c6eb5a89437068d8c6a4accf8031b67](https://github.com/cockroachdb/cockroach/commits/9354770c7c6eb5a89437068d8c6a4accf8031b67):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC2656022987
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-16T09:00:12.966117Z/1029797357098958849/0 @1734339612.982246094,0): "sql txn" meta={id=12ce69db key=/Table/13/1/2024-12-16T08:55:12.665817Z iso=Serializable pri=0.06614327 epo=0 ts=1734339654.981666159,2 min=1734339330.648172842,0 seq=6000} lock=true stat=PENDING rts=1734339330.648172842,0 wto=false gul=1734339331.148172842,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 16, 9, 55, 12, 665816999, time.UTC)
        	            	actual  : time.Date(2024, time.December, 16, 8, 55, 12, 665816999, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-16 09:55:12.665816999 +0000 UTC m=+3633.206536490
        	            	+(time.Time) 2024-12-16 08:55:12.665816999 +0000 UTC m=+33.206536490
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC2656022987
--- FAIL: TestLogGC (351.32s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/2e7361fb-ab01-4290-8567-1fdc8b131565?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [a653b4e4e6483cec7d65808ba4d55d8c63747a6e](https://github.com/cockroachdb/cockroach/commits/a653b4e4e6483cec7d65808ba4d55d8c63747a6e):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC757096706
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-17T08:58:55.507166Z/1030080218800848897/0 @1734425935.523025315,0): "sql txn" meta={id=703f3c95 key=/Table/13/1/2024-12-17T08:53:55.345233Z iso=Serializable pri=0.03589536 epo=0 ts=1734425978.050592498,2 min=1734425653.699656911,0 seq=6000} lock=true stat=PENDING rts=1734425653.699656911,0 wto=false gul=1734425654.199656911,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 17, 9, 53, 55, 345232832, time.UTC)
        	            	actual  : time.Date(2024, time.December, 17, 8, 53, 55, 345232832, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-17 09:53:55.345232832 +0000 UTC m=+3668.026237180
        	            	+(time.Time) 2024-12-17 08:53:55.345232832 +0000 UTC m=+68.026237180
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC757096706
--- FAIL: TestLogGC (355.11s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/ecd927f7-fdb9-4756-a21c-37e0a0681ca6?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [d8de5b82dbbc107e15e59264cd8a48d4b98135cb](https://github.com/cockroachdb/cockroach/commits/d8de5b82dbbc107e15e59264cd8a48d4b98135cb):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC4179068880
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-18T08:51:16.29354Z/1030361829571592193/0 @1734511876.310013795,0): "sql txn" meta={id=ba9234fb key=/Table/13/1/2024-12-18T08:46:16.053219Z iso=Serializable pri=0.00918293 epo=0 ts=1734511907.651712952,2 min=1734511593.154343661,0 seq=6000} lock=true stat=PENDING rts=1734511593.154343661,0 wto=false gul=1734511593.654343661,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 18, 9, 46, 16, 53218955, time.UTC)
        	            	actual  : time.Date(2024, time.December, 18, 8, 46, 16, 53218955, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-18 09:46:16.053218955 +0000 UTC m=+3665.886990150
        	            	+(time.Time) 2024-12-18 08:46:16.053218955 +0000 UTC m=+65.886990150
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC4179068880
--- FAIL: TestLogGC (344.17s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/d4eacc64-6ce7-4310-8bd6-bef27d1d63a4?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [6eb6d4935baf41d60ba3667ed6b6c4ada25cdc26](https://github.com/cockroachdb/cockroach/commits/6eb6d4935baf41d60ba3667ed6b6c4ada25cdc26):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC1001943035
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-20T08:53:56.815999Z/1030928586616078337/0 @1734684836.833858162,0): "sql txn" meta={id=e2ecb8fc key=/Table/13/1/2024-12-20T08:48:57.89798Z iso=Serializable pri=0.00596051 epo=0 ts=1734684897.648156122,2 min=1734684559.220467338,8 seq=6000} lock=true stat=PENDING rts=1734684559.220467338,8 wto=false gul=1734684559.720467338,8
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 20, 9, 48, 57, 897979693, time.UTC)
        	            	actual  : time.Date(2024, time.December, 20, 8, 48, 57, 897979693, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-20 09:48:57.897979693 +0000 UTC m=+3665.288888151
        	            	+(time.Time) 2024-12-20 08:48:57.897979693 +0000 UTC m=+65.288888151
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC1001943035
--- FAIL: TestLogGC (371.06s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/758339fa-94ce-46d6-a04e-0085be193c8e?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [0806ee44e36b62dce75175702b5bfb3db03e0577](https://github.com/cockroachdb/cockroach/commits/0806ee44e36b62dce75175702b5bfb3db03e0577):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC1179909390
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-21T09:01:22.934468Z/1031213163966267393/0 @1734771682.948955235,0): "sql txn" meta={id=9bf3d588 key=/Table/13/1/2024-12-21T08:56:22.880527Z iso=Serializable pri=0.02970188 epo=0 ts=1734771703.125372055,2 min=1734771399.834337580,0 seq=6000} lock=true stat=PENDING rts=1734771399.834337580,0 wto=false gul=1734771400.334337580,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 21, 9, 56, 22, 880526888, time.UTC)
        	            	actual  : time.Date(2024, time.December, 21, 8, 56, 22, 880526888, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-21 09:56:22.880526888 +0000 UTC m=+3635.387795051
        	            	+(time.Time) 2024-12-21 08:56:22.880526888 +0000 UTC m=+35.387795051
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC1179909390
--- FAIL: TestLogGC (330.26s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/ea417064-a702-4340-85f3-4f170241b1d8?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [01f9e61532862cbbdbc64180013ca1cb57f4a017](https://github.com/cockroachdb/cockroach/commits/01f9e61532862cbbdbc64180013ca1cb57f4a017):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC495450581
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-22T08:58:54.927311Z/1031495794510200833/0 @1734857934.946117367,0): "sql txn" meta={id=1fc83334 key=/Table/13/1/2024-12-22T08:53:55.102128Z iso=Serializable pri=0.00926638 epo=0 ts=1734857989.693947009,2 min=1734857655.690603949,0 seq=6000} lock=true stat=PENDING rts=1734857655.690603949,0 wto=false gul=1734857656.190603949,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 22, 9, 53, 55, 102127849, time.UTC)
        	            	actual  : time.Date(2024, time.December, 22, 8, 53, 55, 102127849, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-22 09:53:55.102127849 +0000 UTC m=+3666.249350442
        	            	+(time.Time) 2024-12-22 08:53:55.102127849 +0000 UTC m=+66.249350442
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC495450581
--- FAIL: TestLogGC (367.81s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/540da74c-cbcd-4346-9364-a2d4b4136451?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [bea204ff6caead5a3b47db83d68f2e6e0231b84d](https://github.com/cockroachdb/cockroach/commits/bea204ff6caead5a3b47db83d68f2e6e0231b84d):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC3036772637
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-23T08:58:07.752731Z/1031778755437625345/0 @1734944287.768201230,0): "sql txn" meta={id=a7c45a1c key=/Table/13/1/2024-12-23T08:53:07.908795Z iso=Serializable pri=0.05913586 epo=0 ts=1734944340.742788634,2 min=1734944007.534085827,0 seq=6000} lock=true stat=PENDING rts=1734944007.534085827,0 wto=false gul=1734944008.034085827,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 23, 9, 53, 7, 908795270, time.UTC)
        	            	actual  : time.Date(2024, time.December, 23, 8, 53, 7, 908795270, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-23 09:53:07.90879527 +0000 UTC m=+3664.636423388
        	            	+(time.Time) 2024-12-23 08:53:07.90879527 +0000 UTC m=+64.636423388
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC3036772637
--- FAIL: TestLogGC (365.57s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/90089489-5d0d-4e1a-b415-ca133652a32d?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [d58f071217505b226a097d39f87072f4e1bd8e06](https://github.com/cockroachdb/cockroach/commits/d58f071217505b226a097d39f87072f4e1bd8e06):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC3004143313
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-24T09:03:05.760038Z/1032062847467159553/0 @1735030985.775182126,0): "sql txn" meta={id=d314c122 key=/Table/13/1/2024-12-24T08:58:05.636275Z iso=Serializable pri=0.05494682 epo=0 ts=1735031011.288613797,2 min=1735030702.902947305,0 seq=6000} lock=true stat=PENDING rts=1735030702.902947305,0 wto=false gul=1735030703.402947305,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 24, 9, 58, 5, 636274637, time.UTC)
        	            	actual  : time.Date(2024, time.December, 24, 8, 58, 5, 636274637, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-24 09:58:05.636274637 +0000 UTC m=+3635.703207944
        	            	+(time.Time) 2024-12-24 08:58:05.636274637 +0000 UTC m=+35.703207944
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC3004143313
--- FAIL: TestLogGC (335.16s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/34a95f39-992f-449a-863b-e1645104d0e2?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [270ba1e8fe656fab0a182643fb77a6a5be64c1b0](https://github.com/cockroachdb/cockroach/commits/270ba1e8fe656fab0a182643fb77a6a5be64c1b0):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC3905970931
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-25T09:33:12.851074Z/1032351884464750593/0 @1735119192.866758899,0): "sql txn" meta={id=9685e138 key=/Table/13/1/2024-12-25T09:28:12.929691Z iso=Serializable pri=0.00659935 epo=0 ts=1735119235.453951294,2 min=1735118912.625607717,0 seq=6000} lock=true stat=PENDING rts=1735118912.625607717,0 wto=false gul=1735118913.125607717,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 25, 10, 28, 12, 929690729, time.UTC)
        	            	actual  : time.Date(2024, time.December, 25, 9, 28, 12, 929690729, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-25 10:28:12.929690729 +0000 UTC m=+3639.833885329
        	            	+(time.Time) 2024-12-25 09:28:12.929690729 +0000 UTC m=+39.833885329
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC3905970931
--- FAIL: TestLogGC (352.76s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/62dea4e6-05b2-427c-b526-43e8e3634e57?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [a0faa4e779ac9e54d17e8400141b0aa472887974](https://github.com/cockroachdb/cockroach/commits/a0faa4e779ac9e54d17e8400141b0aa472887974):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC2699719402
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-27T09:30:56.522776Z/1032917668793384961/0 @1735291856.541346302,0): "sql txn" meta={id=c9295a11 key=/Table/13/1/2024-12-27T09:25:57.335504Z iso=Serializable pri=0.01225453 epo=0 ts=1735291967.126387157,2 min=1735291587.784997580,0 seq=6000} lock=true stat=PENDING rts=1735291587.784997580,0 wto=false gul=1735291588.284997580,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 27, 10, 25, 57, 335504139, time.UTC)
        	            	actual  : time.Date(2024, time.December, 27, 9, 25, 57, 335504139, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-27 10:25:57.335504139 +0000 UTC m=+3648.033300970
        	            	+(time.Time) 2024-12-27 09:25:57.335504139 +0000 UTC m=+48.033300970
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC2699719402
--- FAIL: TestLogGC (421.36s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/52e9fbb6-7d01-4f57-833e-7116086bf158?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [58422b11aa3ceb11dd5a382a2236d985517e1506](https://github.com/cockroachdb/cockroach/commits/58422b11aa3ceb11dd5a382a2236d985517e1506):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC2231260287
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-28T09:17:00.203489Z/1033198043860631553/0 @1735377420.221510680,0): "sql txn" meta={id=0fcd76b2 key=/Table/13/1/2024-12-28T09:12:00.257244Z iso=Serializable pri=0.00185589 epo=0 ts=1735377463.783675218,2 min=1735377139.780294381,0 seq=6000} lock=true stat=PENDING rts=1735377139.780294381,0 wto=false gul=1735377140.280294381,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 28, 10, 12, 0, 257243513, time.UTC)
        	            	actual  : time.Date(2024, time.December, 28, 9, 12, 0, 257243513, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-28 10:12:00.257243513 +0000 UTC m=+3664.511070162
        	            	+(time.Time) 2024-12-28 09:12:00.257243513 +0000 UTC m=+64.511070162
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC2231260287
--- FAIL: TestLogGC (357.17s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/1648d070-da4e-4764-b964-02035fda0e66?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [e5ef95eead313b215d8acec85a9ed124d6a1a193](https://github.com/cockroachdb/cockroach/commits/e5ef95eead313b215d8acec85a9ed124d6a1a193):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC2264250705
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-30T09:41:26.460538Z/1033769079532060673/0 @1735551686.478622404,0): "sql txn" meta={id=8376c600 key=/Table/13/1/2024-12-30T09:36:26.51007Z iso=Serializable pri=0.01246044 epo=0 ts=1735551728.187547960,2 min=1735551405.184287461,0 seq=6000} lock=true stat=PENDING rts=1735551405.184287461,0 wto=false gul=1735551405.684287461,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 30, 10, 36, 26, 510070398, time.UTC)
        	            	actual  : time.Date(2024, time.December, 30, 9, 36, 26, 510070398, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-30 10:36:26.510070398 +0000 UTC m=+3658.729625896
        	            	+(time.Time) 2024-12-30 09:36:26.510070398 +0000 UTC m=+58.729625896
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC2264250705
--- FAIL: TestLogGC (354.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/2364f647-7f56-4649-a42d-69307fbef623?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [47699f3887ad5d1b8c7c5905eb5c49628aa59bbe](https://github.com/cockroachdb/cockroach/commits/47699f3887ad5d1b8c7c5905eb5c49628aa59bbe):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC3914000151
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2024-12-31T09:04:42.185412Z/1034044972078628865/0 @1735635882.202174434,0): "sql txn" meta={id=8cc0ad08 key=/Table/13/1/2024-12-31T08:59:42.526087Z iso=Serializable pri=0.00049924 epo=0 ts=1735635951.304760162,2 min=1735635605.276751497,0 seq=6000} lock=true stat=PENDING rts=1735635605.276751497,0 wto=false gul=1735635605.776751497,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2024, time.December, 31, 9, 59, 42, 526087241, time.UTC)
        	            	actual  : time.Date(2024, time.December, 31, 8, 59, 42, 526087241, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2024-12-31 09:59:42.526087241 +0000 UTC m=+3670.000784592
        	            	+(time.Time) 2024-12-31 08:59:42.526087241 +0000 UTC m=+70.000784592
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC3914000151
--- FAIL: TestLogGC (383.11s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/027c98d1-a9e3-4c3a-8b1f-14aad90ec98d?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [47699f3887ad5d1b8c7c5905eb5c49628aa59bbe](https://github.com/cockroachdb/cockroach/commits/47699f3887ad5d1b8c7c5905eb5c49628aa59bbe):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC4291173571
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2025-01-01T09:25:19.785916Z/1034332142970601473/0 @1735723519.803425058,0): "sql txn" meta={id=94df1f80 key=/Table/13/1/2025-01-01T09:20:19.789316Z iso=Serializable pri=0.00596181 epo=0 ts=1735723555.560044960,2 min=1735723238.429179289,0 seq=6000} lock=true stat=PENDING rts=1735723238.429179289,0 wto=false gul=1735723238.929179289,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2025, time.January, 1, 10, 20, 19, 789315587, time.UTC)
        	            	actual  : time.Date(2025, time.January, 1, 9, 20, 19, 789315587, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2025-01-01 10:20:19.789315587 +0000 UTC m=+3636.899412826
        	            	+(time.Time) 2025-01-01 09:20:19.789315587 +0000 UTC m=+36.899412826
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC4291173571
--- FAIL: TestLogGC (345.85s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server.TestLogGC [failed](https://tanzanite.cluster.engflow.com/invocations/default/ebe5ed8a-96fd-46b2-96fd-2251a37d2a1a?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyOnNlcnZlcl90ZXN0) on master @ [fb01a96ccf75e62f6ecf6c7b1e3ada7453a76b0c](https://github.com/cockroachdb/cockroach/commits/fb01a96ccf75e62f6ecf6c7b1e3ada7453a76b0c):


```
=== RUN   TestLogGC
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogGC540995508
    test_log_scope.go:76: use -show-logs to present logs inline
    server_systemlog_gc_test.go:130: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:130
        	Error:      	Received unexpected error:
        	            	TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh due to encountered recently written committed value /Table/13/1/2025-01-02T08:47:13.131992Z/1034607765575794689/0 @1735807633.147216171,0): "sql txn" meta={id=5bb5960a key=/Table/13/1/2025-01-02T08:42:12.88234Z iso=Serializable pri=0.01338106 epo=0 ts=1735807633.360920900,2 min=1735807348.862616795,0 seq=6000} lock=true stat=PENDING rts=1735807348.862616795,0 wto=false gul=1735807349.362616795,0
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:131: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:131
        	Error:      	Not equal: 
        	            	expected: time.Date(2025, time.January, 2, 9, 42, 12, 882339954, time.UTC)
        	            	actual  : time.Date(2025, time.January, 2, 8, 42, 12, 882339954, time.UTC)
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,2 @@
        	            	-(time.Time) 2025-01-02 09:42:12.882339954 +0000 UTC m=+3634.571578063
        	            	+(time.Time) 2025-01-02 08:42:12.882339954 +0000 UTC m=+34.571578063
        	            	 
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:132: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:132
        	Error:      	Should be true
        	Test:       	TestLogGC
        	Messages:   	Expected rowsGCd >= 2000, found 0
    server_systemlog_gc_test.go:133: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:133
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2000
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:141: 
        	Error Trace:	pkg/server/server_systemlog_gc_test.go:141
        	Error:      	Not equal: 
        	            	expected: 5
        	            	actual  : 2005
        	Test:       	TestLogGC
    server_systemlog_gc_test.go:142: -- test log scope end --
test logs left over in: outputs.zip/logTestLogGC540995508
--- FAIL: TestLogGC (310.11s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogGC.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/server/server_systemlog_gc_test.go
+++ b/pkg/server/server_systemlog_gc_test.go
@@ -67,7 +67,7 @@ func TestLogGC(t *testing.T) {
 			a.NoError(err)
 		}
 	}
-	maxTS1 := timeutil.Now()
+	maxTS1 := timeutil.Now().Add(time.Duration(-24*7) * time.Hour)
 	maxTS2 := maxTS1.Add(time.Second)
 	maxTS3 := maxTS2.Add(time.Second)
 	maxTS4 := maxTS3.Add(time.Second)

